src/main/java/com/google/sps/data/LearningPathService.java

```diff
@@ -95,6 +96,21 @@ private void storeItem(LearningItem item, long learningSectionId, long learningP
 		datastore.put(e);
 	}
 
+    private void storeFeedback(ItemFeedback feedback) {
+		storeFeedback(feedback, feedback.getLearningItem(), feedback.getLearningPath());
+	}
+
+	private void storeFeedback(ItemFeedback item, long learningItemId, long learningPathId) {
+		Entity e = new Entity(ITEM_FEEDBACK, item.getId());
+		e.setProperty("learningPath", learningPathId);
+		e.setProperty("learningItem", learningItemId);
+		e.setProperty("userId", item.getUserId());
+		e.setProperty("rating", item.getRating());
+		e.setProperty("completed", item.isCompleted());
+        e.setProperty("id", item.getId());
+		datastore.put(e);
+	}
+
```

Comment:
This looks similar to `submitFeedback`, which handles adding and updating ItemFeedback.

NOTE: Looks like there's a bug in `submitFeedback` where we don't properly update the ItemFeedback (we are missing a `datastore.put(..)` after updating the rating).

I think those are the only two scenarios for updating a `ItemFeedback`. Did you have another use in mind for `storeFeedback(..)`?